add-on/chrome/bootstrap.js

```diff
@@ -211,6 +211,11 @@ var WindowListener = {
             });
           };
 
+          if (PrivateBrowsingUtils.isWindowPrivate(window)) {
+            resolve(null);
+            return;
+          }
```

Comment:
Although, it's quite possible the only uses of the `Promise` is for testing, e.g., https://dxr.mozilla.org/mozilla-central/source/browser/components/uitour/test/head.js#167-174 to indicate the test should continue. We should have a mochitest similar to https://dxr.mozilla.org/mozilla-central/source/browser/components/uitour/test/browser_showMenu_controlCenter.js to make sure the panel opens in non-private browsing and doesn't in private browsing.